config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'news_feed/index'
```

Comment:
Not necessarily needed for this PR, but at some point you should namespace this to a user's newsfeed. Which means the url should be along the lines of `/user/:user_id/news_feed/index` where `:user_id` is a user id.

Otherwise at the moment this is only one newsfeed for everyone